politeiawww/api/www/v1/v1.go

```diff
@@ -209,6 +209,7 @@ const (
 	ErrorStatusInvalidInvoiceStatusTransition ErrorStatusT = 65
 	ErrorStatusReasonNotProvided              ErrorStatusT = 66
 	ErrorStatusInvoiceDuplicate               ErrorStatusT = 67
+	ErrorStatusInvoiceNotEditable             ErrorStatusT = 68
```

Comment:
Would prefer a more generic `ErrorStatusWrongInvoiceStatus` that could be used in other situations where an invoice has an incorrect status.  This would also be consistent with how pi treats it.